net.sonic_pi.SonicPi.json

```diff
@@ -150,14 +161,42 @@
                 }
             ]
         },
+        {
+            "name": "cmake",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Kitware/CMake/releases/download/v3.24.2/cmake-3.24.2.tar.gz",
+                    "sha256": "0d9020f06f3ddf17fb537dc228e1a56c927ee506b486f55fe2dc19f69bf0c8db"
+                }
+            ]
+        },
         {
             "name": "sonicpi",
             "buildsystem": "simple",
             "build-commands": [
-                "cd app && ./linux-prebuild.sh --build-aubio",
-                "cd app && ./linux-config.sh --config RelWithDebInfo",
-                "cmake --build app/build"
+                /* make downloaded vcpkg binary executable */
+                "chmod +x app/vcpkg/vcpkg",
```

Comment:
except we want to build on aarch64, so this is NO NO.

Reply:
I honestly didn't know that other architectures than x86 are supported, obviously this can't be done this way. It can be built from source and I was planning to do that change, I just took the shortcut and left it for later to deal first with the elixir part, which is the one I wasn't sure was possible at all. I'll fix this.